android/src/main/java/com/reactnativecommunity/cameraroll/CameraRollModule.java

```diff
@@ -1088,4 +1088,7 @@ protected void doInBackgroundGuarded(Void... params) {
   public void getPhotoThumbnail(String internalID, ReadableMap options, Promise promise) {
     promise.reject("CameraRoll:getPhotoThumbnail", "getPhotoThumbnail is not supported on Android");
   }
+
+  public void addListener(String eventName) {}
+  public void removeListeners(double count) {}
```

Comment:
Just trying newarch on 0.74 and for me this needs to be `int` and not `double`

Reply:
Create a PR and will merge it 🙂